src/llmcompressor/transformers/sparsification/sparse_model.py

```diff
@@ -50,16 +45,6 @@ def from_pretrained(
         *model_args,
         **kwargs,
     ) -> PreTrainedModel:
-        """
```

Comment:
Why remove this docstring? Might want to include the other docstring in here as well